docs/cloud/getting-started-with-airbyte-cloud.md

```diff
@@ -208,12 +208,6 @@ Depending on your [data residency](https://docs.airbyte.com/cloud/managing-airby
 
 ### European Union
 
-:::note 
-
-Some workflows still run in the US, even when the data residency is in the EU. If you use the EU as a data residency, you must allowlist the following IP addresses from both GCP us-west3 and AWS eu-west-3.
-
-:::
-
 #### GCP region: us-west3
```

Comment:
@malikdiarra could we also remove the US-west3 IP list from this section then, it should no longer be needed either?

Reply:
Done!